changedetectionio/templates/settings.html

```diff
@@ -83,7 +83,18 @@
                     </span>
                     </fieldset>
 
-
+                    <fieldset class="pure-group">
+                      {{ render_field(form.global_subtractive_selectors, rows=5, placeholder="header
+footer
+nav
+.stockticker") }}
+                      <span class="pure-form-message-inline">
+                      <br/>
+                      Ignore HTML tags or matches to CSS selectors.
```

Comment:
This should be "Remove HTML element by CSS selector before text conversion"

Reply:
Also better is to wrap this like 
```
<UL><LI>    Ignore HTML tags or matches to CSS selectors.</li>
<LI>Add one tag or CSS selector per line to ignore multiple parts of the HTML.</LI>
...
```

same like the other existing form entries